sympy/polys/tests/test_densetools.py

```diff
@@ -282,6 +292,32 @@ def test_dmp_revert():
     raises(MultivariatePolynomialError, lambda: dmp_revert([[1]], 2, 1, QQ))
 
 
+def test_dup_series_reversion(trials=50):
+    for _ in range(trials):
+        deg = randint(2, 10)
+        f = _dup_random(-10, 10, deg, QQ)
+        if len(f) < 2:
+            continue
+
+        f[-1] = QQ(0)
+        if f[-2] == 0:
+            f[-2] = QQ(1, 1)
+
+        g = dup_series_reversion(f, deg, QQ)
```

Comment:
In all of the cases the same `deg` is used for `_dup_random` and `dup_series_reversion`. I think it is good to have cases where they are not equal as well.